### ISSUE TITLE
createImageBitmap() `imageOrientation` option type incorrect in 5.0.0-beta

### ISSUE THREAD

**Issue by frank-weindel** [microsoft/TypeScript#53053]
# lib Update Request

## Configuration Check

My compilation *target* is `ES2022` and my *lib* is `the default`.

## Missing / Incorrect Definition

In 5.0.0-beta the DOM provided [createImageBitmap](https://developer.mozilla.org/en-US/docs/Web/API/createImageBitmap) (MDN linked) function is not allowing the string "none" to be set on the `imageOrientation` option property.

It should be allowing the strings `"flipY" | "none"` according to MDN (and does work in previous TS versions) but in 5.0.0-beta the only strings allowed are: `"flipY" | "from-image"`.

`"from-image"` seems to be coming from a related but separate CSS property called [`image-orientation`](https://developer.mozilla.org/en-US/docs/Web/CSS/image-orientation).

**node_modules/typescript/lib/lib.dom.d.ts**:
```ts
// ...
interface ImageBitmapOptions {
    colorSpaceConversion?: ColorSpaceConversion;
    imageOrientation?: ImageOrientation;
    premultiplyAlpha?: PremultiplyAlpha;
    resizeHeight?: number;
    resizeQuality?: ResizeQuality;
    resizeWidth?: number;
}
// ...
declare function createImageBitmap(image: ImageBitmapSource, options?: ImageBitmapOptions): Promise<ImageBitmap>;
declare function createImageBitmap(image: ImageBitmapSource, sx: number, sy: number, sw: number, sh: number, options?: ImageBitmapOptions): Promise<ImageBitmap>;
// ...
type ImageOrientation = "flipY" | "from-image";
// ...
```

## Sample Code

Sample:
```ts
createImageBitmap({} as ImageBitmap, {
    imageOrientation: 'none'
 // ^^^^^^^^^^^^^^^^ Type '"none"' is not assignable to type 'ImageOrientation | undefined'.(2322)
});
```

[Playground Link](https://www.typescriptlang.org/play?target=9&ts=5.0.0-beta#code/MYJwpghgLmCSC2EDmYBCBLKiAOAKA3gL4AEEAzsQsmpjgDTH4BQxrx6iKA8iOmAHZRo6APb8AXMQDk-MWClNCASgDcTJkA)

## Documentation Link

https://developer.mozilla.org/en-US/docs/Web/API/createImageBitmap

**Comment by Josh-Cena** [comment]
Hi, there is a change to the `createImageBitmap` API that MDN is not catching up with (and even seems to be unfinished within the standard group). See https://github.com/mdn/content/issues/23564, https://github.com/mdn/content/issues/24857. https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1507 also has some more information.

**Comment by jakebailey** [comment]
This was fixed in #53652.
